src/comp_loinc/loinc_builder_steps.py

```diff
@@ -366,22 +382,10 @@ def loinc_part_hierarchy_all(self):
                 self.runtime.current_module.add_entity(child_part)
 
             edge: Edge
+            subclassable_edges = [LoincPartEdge.parent_comp_by_system, LoincDanglingNlpEdges.nlp_parent]
             for edge in child_part_node.get_all_out_edges():
-                if edge.edge_type.type_ is LoincPartEdge.parent_comp_by_system:
-                    parent_part_node = edge.to_node
-                    parent_part_number = parent_part_node.get_property(
-                        type_=LoincPartProps.part_number
-                    )
-
-                    parent_part = self.runtime.current_module.get_entity(
-                        entity_id=parent_part_number, entity_class=LoincPart
-                    )
-                    if parent_part is None:
-                        parent_part = LoincPart(id=parent_part_number)
-                        self.runtime.current_module.add_entity(parent_part)
-
-                    if parent_part.id not in child_part.sub_class_of:
-                        child_part.sub_class_of.append(parent_part.id)
+                if edge.edge_type.type_ in subclassable_edges:
+                    self._edge_to_subclassof(child_part, edge)
```

Comment:
Addition to `LoincBuilderSteps.loinc_part_hierarchy_all()`.

Now, these edges are being processed here:
```
subclassable_edges = [LoincPartEdge.parent_comp_by_system, LoincDanglingNlpEdges.nlp_parent]
```

Was previously just `LoincPartEdge.parent_comp_by_system`.

I'm wondering if `LoincTreeEdges.tree_parent` is also supposed to be here? Maybe if this is still an issue, that will solve it?:
- #99